src/Controls/src/Core/HandlerImpl/Window/Window.cs

```diff
@@ -6,26 +6,20 @@ namespace Microsoft.Maui.Controls
 {
 	public partial class Window
 	{
+		[Obsolete("Use WindowHandler.Mapper instead.")]
 		public static IPropertyMapper<IWindow, WindowHandler> ControlsWindowMapper =
-			new PropertyMapper<IWindow, WindowHandler>(WindowHandler.Mapper);
+			new PropertyMapper<Window, WindowHandler>(WindowHandler.Mapper);
 
-		// ControlsWindowMapper incorrectly typed to WindowHandler
-		static IPropertyMapper<IWindow, IWindowHandler> Mapper =
-			new PropertyMapper<IWindow, IWindowHandler>(ControlsWindowMapper)
-			{
+		internal static void RemapForControls()
+		{
 #if ANDROID
-				// This property is also on the Application Mapper since that's where the attached property exists				
-				[PlatformConfiguration.AndroidSpecific.Application.WindowSoftInputModeAdjustProperty.PropertyName] = MapWindowSoftInputModeAdjust,
+			// This property is also on the Application Mapper since that's where the attached property exists
+			WindowHandler.Mapper.ReplaceMapping<IWindow, IWindowHandler>(PlatformConfiguration.AndroidSpecific.Application.WindowSoftInputModeAdjustProperty.PropertyName, MapWindowSoftInputModeAdjust);
 #endif
 
 #if WINDOWS
-				[nameof(ITitledElement.Title)] = MapWindowTitle
+			WindowHandler.Mapper.ReplaceMapping<Window, IWindowHandler>(nameof(ITitledElement.Title), MapTitle);
```

Comment:
Should this be `PrependToMapping` or `ModifyToMapping` and then MapTitle invokes the action?

Reply:
It was... they your suggested we replace the whole method because it calls the same `UpdateTitle` method - just with a different context. I previously had it as `AppendToMapping`.

If we have it as prepend to mapping, then the title will always be the Core implementation since that will override whatever we add in Controls?

Reply:
Done! And added a test to prove this!